pyomo/contrib/pynumero/interfaces/tests/test_dynamic_model.py

```diff
@@ -368,9 +368,12 @@ def max_outflow(m, t):
 
     return m2
 
+
 class TestGreyBoxModel(unittest.TestCase):
 
-    @unittest.skipIf(not ipopt_available, "CyIpopt needed to run tests with solve")
+    @unittest.skipIf(
+        not pyo.SolverFactory('ipopt').available(exception_flag=False),
```

Comment:
Is this one actually `ipopt` or is it supposed to be `cyipopt`?

Reply:
This is really`ipopt`: the test is using the `'ipopt'` solver and not `'cyipopt'`